bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -178,6 +178,39 @@ spec:
             command: ["test", "-f", "/etc/cni/net.d/80-openshift-network.conf"]
           initialDelaySeconds: 5
           periodSeconds: 5
+{{- if .EnableConnectivityChecks }}
+      - name: sdn-check-endpoints
+        image: quay.io/openshift/origin-cluster-kube-apiserver-operator:latest
+        imagePullPolicy: IfNotPresent
+        terminationMessagePolicy: FallbackToLogsOnError
+        command:
+          - cluster-kube-apiserver-operator
+          - check-endpoints
+        args:
+          - --listen
+          - 0.0.0.0:17698
+          - --namespace
+          - $(POD_NAMESPACE)
+          - --v
+          - '2'
+        env:
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+        ports:
+          - name: check-endpoints
+            containerPort: 17698
```

Comment:
Since this is a `hostNetwork` pod, this is a host port. However, this isn't in one of the open ranges (9000-9999, 30000-32767) so it will be blocked between nodes by default....

It seems like possibly in cluster-kube-apiserver-operator, the listening port only gets used for the health checks (from the local node), and the declared `check-endpoints` port is not actually used? In which case since we are not doing health checks on these containers, the port (and the `--listen`) may be unnecessary.

(If we _are_ grabbing a host port, even if just for local health checks, it will need to be registered in https://github.com/openshift/enhancements/blob/master/enhancements/network/host-port-registry.md)